examples/read_bids_datasets.py

```diff
@@ -0,0 +1,93 @@
+"""
+=========================================
+02. Read and load datasets in BIDS format
+=========================================
+
+When working with neural data in the BIDS format, we usually have
+varying types of data, which can be loaded in via `read_raw_bids` function.
+
+- MEG
+- EEG (scalp)
+- iEEG (ECoG and SEEG)
+- the anatomical MRI scan of a study participant
+
+In order to check if your dataset is compliant with BIDS, you can first
+run the BIDS validator online.
+
+In this tutorial, we show how `read_raw_bids`
+can be used to load any BIDS format data,
+and to display the data that is saved within the accompanying sidecars.
+
+"""
+# Authors: Adam Li <https://github.com/adam2392>
```

Comment:
can you update this to be your email address? The commits are signed by email address anyway ...